### ISSUE TITLE
deb install fails on raspbian due to openjdk 9 upgrade

### ISSUE THREAD

**Issue by astfgl** [thingsboard/thingsboard#706]
Raspbian has upgraded to openjdk-9-jre and openjdk-9-jre-headless, so the 1.4 deb is failing on install:

```
# dpkg -l | grep jdk
ii  openjdk-9-jre-headless:armhf    9~b181-4~bpo9+rpt1           armhf        OpenJDK Java runtime, using Hotspot JIT (headless)

# dpkg -i thingsboard-1.4.deb 
Selecting previously unselected package thingsboard.
(Reading database ... 35872 files and directories currently installed.)
Preparing to unpack thingsboard-1.4.deb ...
Adding group `thingsboard' (GID 115) ...
Done.
Unpacking thingsboard (1.4.0-1) ...
dpkg: dependency problems prevent configuration of thingsboard:
 thingsboard depends on openjdk-8-jre | java8-runtime | oracle-java8-installer | openjdk-8-jre-headless; however:
  Package openjdk-8-jre is not installed.
  Package java8-runtime is not installed.
  Package oracle-java8-installer is not installed.
  Package openjdk-8-jre-headless is not installed.

dpkg: error processing package thingsboard (--install):
 dependency problems - leaving unconfigured
Processing triggers for systemd (232-25+deb9u2) ...
Errors were encountered while processing:
 thingsboard
```

**Comment by mp-loki** [comment]
The error is self-explanatory - it required Java 8 to run

**Comment by astfgl** [comment]
Perhaps I did not explain myself properly. Version 8 of openjdk was removed from raspbian yesterday and replaced with version 9.  My issue was to ask the package maintainers to bump the pre-requesite package version up to include version 9, so that the thingsboard deb package would be installable.

It looks like there was a raspbian package management error. For all of yesterday, the only openjdk packages available were version 9. Today I can see that version 8 is back as well as version 9.

The issue still stands, as I don't know how long the version 8 packages will remain available now that version 9 is in the repository. I believe it would be prudent to start testing with version 9 so that there is an upgrade path when version 8 is intentionally removed.
